package.json

```diff
@@ -14,7 +14,7 @@
     "url": "https://github.com/twilio/twilio-node.git"
   },
   "dependencies": {
-    "request": "2.55.x",
+    "request": "2.70.x",
```

Comment:
Haha true, I was reviewing the history of some package.json and I mixed up the final version to set.